package.json

```diff
@@ -5,6 +5,7 @@
     "description": "CommCare HQ Javascript Libraries",
     "private": true,
     "dependencies": {
+        "@babel/core": "7.25.2",
```

Comment:
I think this along with most other depednencies added in 1a54bb20a479e692f741c6da3acd77eafb045643 should be added to `devDependencies`.

Reply:
i originally had the dependencies in dev dependencies, but it seems like we do not install dev dependencies on production machines. The build failed because the dependencies were not installed. Do we want to change this policy?

Reply:
> this can be updated later based on what the team decides

Commenting because we should avoid the "we'll talk about this later" approach as much as possible, and I don't think this needs to be an open question - I think it's preferable to put webpack in prod dependencies, as is done with requirejs, rather than installing dev dependencies on the prod servers.

Reply:
Fair enough, at least until we move JS bundling from the deploy to something like Github Actions.